scripts/capture_lesson.py

```
@@ -0,0 +1,256 @@
+        os.path.join("build", "lessons", "gpu", target_name, "Release", f"{target_name}.exe"),
+        os.path.join("build", "lessons", "gpu", target_name, target_name),
+    ]
+    for path in candidates:
+        if os.path.isfile(path):
+            return path
+    return None
+
+
+def build_lesson(target_name):
+    """Build the lesson with FORGE_CAPTURE enabled."""
+    print(f"Configuring with FORGE_CAPTURE=ON...")
```

Ruff (0.15.0)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)